assembly/src/main/resources/features.xml

```diff
@@ -96,6 +97,27 @@
         <bundle>mvn:org.apache.karaf.cellar/org.apache.karaf.cellar.cloud/${project.version}</bundle>
     </feature>
 
+    <feature name="cellar-kubernetes" description="Kubernetes support in clusters" version="${project.version}" resolver="(obr)">
+        <configfile finalname="/etc/org.apache.karaf.cellar.kubernetes.cfg">
+            mvn:org.apache.karaf.cellar/apache-karaf-cellar/${project.version}/cfg/kubernetes
+        </configfile>
+        <feature version="${project.version}">cellar-core</feature>
+        <feature>cxf-jaxrs</feature>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-core/2.4.1</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-annotations/2.4.1</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-databind/2.4.1</bundle>
+        <bundle>mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.4.1</bundle>
+        <bundle>mvn:javax.validation/validation-api/1.1.0.Final</bundle>
+        <bundle>mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.4.1</bundle>
+        <bundle>mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.4.1</bundle>
+        <bundle>mvn:io.fabric8/fabric-utils/${fabric8.version}</bundle>
+        <bundle>mvn:commons-lang/commons-lang/2.6</bundle>
+        <bundle>wrap:mvn:org.yaml/snakeyaml/1.5</bundle>
+        <bundle>wrap:mvn:org.json/json/20140107</bundle>
+        <bundle>mvn:io.fabric8/kubernetes-api/${fabric8.version}</bundle>
+        <bundle>mvn:org.apache.karaf.cellar/org.apache.karaf.cellar.kubernetes/${project.version}</bundle>
+    </feature>
+
```

Comment:
See upper comment.

Anyway, do you think I should declare all versions in root pom.xml?